backend/src/main/java/com/example/parkingspot/controller/PersonController.java

```diff
@@ -17,6 +18,7 @@
 import com.example.parkingspot.service.CarService;
 import com.example.parkingspot.service.PersonService;
 
+@CrossOrigin
```

Comment:
If we disable cors in SecurityConfig, will this be needed?

Reply:
Correct! This annotation does nothing if cors is disabled